packages/libtreesitter/build.sh

```diff
@@ -7,3 +7,13 @@ TERMUX_PKG_SRCURL=https://github.com/tree-sitter/tree-sitter/archive/refs/tags/v
 TERMUX_PKG_SHA256=b355e968ec2d0241bbd96748e00a9038f83968f85d822ecb9940cbe4c42e182e
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make() {
```

Comment:
Overriding `termux_step_make{,_install}` will cause library and other files to not be included in package anymore. Please run additional steps for building and installing tree-sitter in `termux_step_post_make_install` instead

Reply:
Thanks for the suggestion, I created another commit with the adjustments.